src/styles/box/body.scss

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  width: $card-width;
+  height: 406px;
+  display: flex;
+  align-items: center;
+  flex-flow: column wrap;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  text-align: center;
+  margin: 0;
+  padding: 32px 16px;
+
+  &__bg {
+    width: 160px;
+    height: 134px;
+    background-image: url('../images/imac.jpeg');
+    background-size: cover;
+    background-position: center;
+    margin-left: 3px;
+    margin-bottom: 40px;
+  }
+
+  &__details {
+    width: 166px;
+  }
+
+  &__title {
+    width: 166px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: #060b35;
+    margin: 0;
+  }
+
+  &__code {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+    color: #616070;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    width: 166px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0;
+
+    &__revievs {
```

Comment:
There is a typo in the class name `&__revievs`. It should be `&__reviews` to match the HTML class name.